sketch.js

```diff
@@ -1,137 +1,161 @@
-let shapeClassifier;
-let canvas;
-let img;
-let buttons;
-
-let labels = [
-  "square",
-  "circle",
-  "triangle",
-]
-let label = labels[0];
-let drawing_array = [];
-let drawing = [];
-let i = 0;
+// GLOBALS
+let shapeClassifier;  // ml5.neuralnetwork()
+let canvas;  // createCanvas()
+
+// LABELS AND CYCLING
+let labels = ["square","circle","triangle"];  
+let i = 0;  // MOVE INTO LOCAL
+let current_label = labels[0];  // defaults drawing label to square
+
+// DRAWINGS
+let DRAW = {
+  FREE:[],
```

Comment:
object FREE is the array of vertices composing a free-hand drawing. Clears at the start of every new drawing.